src/app/modules/vault/modules/organization-vault/organization-vault.component.html

```diff
@@ -4,15 +4,15 @@
       <div class="groupings">
         <div class="content">
           <div class="inner-content">
-            <app-vault-filter
+            <app-organization-vault-filter
               #vaultFilter
               [showFolders]="false"
```

Comment:
The `showFolders` input was added for this case (don't show it for orgs). Can we do something like default it to `false` in the org vault filter component? Then we don't have to pass it and that cleans this up a little.

Reply:
Fixed with [[fix] Simplify hiding unwanted filters in the org vault](https://github.com/bitwarden/web/pull/1700/commits/e07deff505d6c3b2b57c3f8121f8908a3581b069)

I tweaked this a bit and also removed these properties completely, similar inputs already existed in jslib.

Reply:
Awesome, yeah that's much better!